src/components/layout/mainNavbar.module.css

```diff
@@ -0,0 +1,56 @@
+.header {
+  width: 100%;
+  background-color: rgb(235, 235, 235);
+  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.2);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  font-family: "Lato", sans-serif;
+  font-size: 2rem;
+  padding-left: 50px;
+}
+
+.header ul {
+  list-style: none;
```

Comment:
What is this?

Reply:
Because I want to hide the dot on the link website.